packages/story-editor/src/components/library/panes/shopping/product.js

```diff
@@ -46,7 +46,11 @@ function Product({ product, onClick, onFocus, isOnPage, canAddMore }) {
         onClick={onClick}
         onFocus={onFocus}
       />
-      <ProductImage product={product} />
+      <ProductImage
+        product={product}
+        isOnPage={isOnPage}
+        canAddMore={canAddMore}
```

Comment:
Nit: it would be more readable to just have one prop, e.g. `isEnabled` or `canDrag` or something